src/EFCore.PG.PGroonga/Query/ExpressionTranslators/Internal/NpgsqlPGroongaMethodTranslator.cs

```diff
@@ -0,0 +1,108 @@
+{
+    /// <summary>
+    /// Provides translations for PGroonga full-text search methods.
+    /// </summary>
+    public class NpgsqlPGroongaMethodTranslator : IMethodCallTranslator
+    {
+        static readonly IReadOnlyDictionary<string, string> SqlNameByMethodName = new Dictionary<string, string>
+        {
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaCommand)] = "pgroonga_command",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaCommandEscapeValue)] = "pgroonga_command_escape_value",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaEscape)] = "pgroonga_escape",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaFlush)] = "pgroonga_flush",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaHighlightHtml)] = "pgroonga_highlight_html",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaIsWritable)] = "pgroonga_is_writable",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaMatchPositionsByte)] = "pgroonga_match_positions_byte",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaMatchPositionsCharacter)] = "pgroonga_match_positions_character",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaNormalize)] = "pgroonga_normalize",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaQueryEscape)] = "pgroonga_query_escape",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaQueryExpand)] = "pgroonga_query_expand",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaQueryExtractKeywords)] = "pgroonga_query_extract_keywords",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaSetWritable)] = "pgroonga_set_writable",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaScore)] = "pgroonga_score",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaSnippetHtml)] = "pgroonga_snippet_html",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaTableName)] = "pgroonga_table_name",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaWalApply)] = "pgroonga_wal_apply",
+            [nameof(NpgsqlPGroongaDbFunctionsExtensions.PgroongaWalTruncate)] = "pgroonga_wal_truncate"
+        };
+
+        /// <inheritdoc />
+        [CanBeNull]
+        public Expression Translate(MethodCallExpression e)
+        {
+            if (e.Method.DeclaringType != typeof(NpgsqlPGroongaDbFunctionsExtensions) &&
+                e.Method.DeclaringType != typeof(NpgsqlPGroongaLinqExtensions))
+                return null;
+
+            if (SqlNameByMethodName.TryGetValue(e.Method.Name, out var sqlFunctionName))
+            {
+                if (sqlFunctionName == "pgroonga_score")
+                {
+                    var column = e.Arguments[1] as ColumnExpression;
+                    return new SqlFunctionExpression(sqlFunctionName, e.Method.ReturnType, new[]
```

Comment:
This function (`pgroonga_score`) requires `tableoid` and `ctid` as the arguments, I didn't find a way to get them, so I write these dirty codes here. I wanna to know what's the best practices to do so.